src/cljs/athens/hotkeys.cljs

```diff
@@ -0,0 +1,75 @@
+(ns athens.hotkeys
+  (:require
+    ["react-hotkeys" :refer [GlobalHotKeys HotKeys configure]]
+    [re-frame.core :refer [dispatch subscribe]]
+    [athens.util :as util]
+    [athens.router :as router]))
+
+(configure #js{:ignoreTags #js[]})
+
+(def modkey
+  (let [os (util/get-os)]
+    (or (and (= os :mac) "meta")
+      (and (= os :windows) "ctrl")
+      (and (= os :linux) "ctrl"))))
+
+(defn with-mod
+  [combination]
+  (str modkey "+" combination))
+
+(defn not-editing
+  []
+  (nil? @(subscribe [:editing/uid])))
+
+(defn selected-items-not-empty
+  []
+  (not-empty @(subscribe [:selected/items])))
+
+(defn maybe-undo-or-redo
```

Comment:
This name confuses me a bit.
Why `maybe-undo-or-redo` when in body we check if not editing or something is selected?
`when-undo-allowed` comes to mind by looking at usage.

Reply:
Felt a bit repetitive, but I had that in mind initially. Sounds good too!